Packs/CommonScripts/Scripts/ModifyDateTime/ModifyDateTime.py

```diff
@@ -0,0 +1,30 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+import dateparser
+
+
+def apply_variation(original_datetime: datetime = None, variation: str = None) -> datetime:
+    new_time = None
+    try:
+        new_time = dateparser.parse(variation, settings={'RELATIVE_BASE': original_datetime})
+    except Exception as err:
+        return_error(f"Error adding variation to the date / time - {err}")
```

Comment:
<img width="1550" alt="Screen Shot 2020-07-16 at 2 36 31" src="https://user-images.githubusercontent.com/57589449/87641042-3c3a8280-c750-11ea-9274-a71e931a4f82.png">
I entered an incorrect variation
and "except Exception" does not catch the error
and the "new_time" returns None

Reply:
I have pushed new changes for checking.